symbols.js

```diff
@@ -395,12 +395,25 @@ const symbols = [
     }
 ];
 
+let symbols = []
+
+try {
+    // Catching invallid json, not an invalid symbol array
+    symbols = JSON.parse(window.localStorage.getItem("symbols"));
+} catch(err) {
+    console.error(err);
+    symbols = []
+}
+if (!symbols) {
+    symbols = symbols_default;
+}
+
 function search(searchTerm) {
     searchTerm = searchTerm?.toLowerCase() ?? "";
 
     return symbols.filter((s) => {
         /* Get hex representation of codepoint, e.g. 00A0 for &nbsp; or 20AC for € */
-        const codePoint = s.glyph.codePointAt(0).toString(16).padStart(4, 0);
+        const codePoint = s.glyph.codePointAt(0)?.toString(16).padStart(4, 0);
```

Comment:
Could we make it mandatory that a symbol have a code point at the 0th position? As in, if they hit enter with an empty text box the symbol either reverts to its default state (for existing symbols) or disappears entirely (for new symbols)?

Reply:
New symbols are created with an empty code point. I can change that to a space (or any other symbol), but then users have to delete the space when editing a new symbol.
Other option is to use a magic symbol, an remove that symbol before the user start editing.  Disadvantages: the magic symbol cannot be used, and extra code needed to check for the magic character.
Adding the question mark is the simplest solution.